src/main/java/org/trigon/utilities/Conversions.java

```diff
@@ -2,6 +2,9 @@
 
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
 
+/**
+ * A class that contains various methods for unit and String conversions.
```

Comment:
there are more conversions here 
give string and unit as examples, but don't limit to it